maintenance/README.md

```diff
@@ -33,24 +33,52 @@ For the sake of simplicity all examples will use jcxjin-test.
    Maintenance mode on.
 
    ```
-   oc patch route pims-app-test -n jcxjin-test -p \
+   oc patch route pims-app-test -n 354028-test -p \
        '{ "spec": { "to": { "name": "proxy-caddy" }, "port": { "targetPort": "2015-tcp" }}}'
-   oc patch route proxy-caddy -n jcxjin-test -p \
+   oc patch route proxy-caddy -n 354028-test -p \
        '{ "spec": { "to": { "name": "pims-app-test" }, "port": { "targetPort": "8080-tcp" }}}'
    ```
 
    Maintenance mode off.
 
    ```
-   oc patch route pims-app-test -n jcxjin-test -p \
+   oc patch route pims-app-test -n 354028-test -p \
        '{ "spec": { "to": { "name": "pims-app-test" }, "port": { "targetPort": "8080-tcp" }}}'
-   oc patch route proxy-caddy -n jcxjin-test -p \
+   oc patch route proxy-caddy -n 354028-test -p \
        '{ "spec": { "to": { "name": "proxy-caddy" }, "port": { "targetPort": "2015-tcp" }}}'
    ```
 
 ### Build and Deployment
 
-This application's template has been broken down into build and deploy components.
+This application's template has been broken down into build and deploy components. The following commands will create all required bc/dc/service/route objects, assuming that the defaults used in the maintenance script are still correct(see below to confirm:
+
+```
+./maintenance.sh tools s2i
+./maintenance.sh tools build
+./maintenance.sh $YOUR_TARGET_ENV deploy
+```
+
+##### S2I Build
+
+Template:
+
+- ../openshift/templates/maintenance/caddy.s2i.bc.yaml
+
+Contains:
+
+- ImageStream
+- BuildConfig
+
+Default vars:
+
+- NAME: s2i-caddy
+- GIT_REPO: https://github.com/BCDevOps/s2i-caddy.git
```

Comment:
This parameter does not appear to be correctly named.  Additionally the description states that this should be our URL, not the default value (unless experimenting).

Reply:
That was true when they were actually hosting this image on openshift. Now since they aren't hosting this image we have to build it ourselves.

Reply:
Shouldn't we update the documentation so we don't even point to the BCGov repo then?

Reply:
We would then need to copy all of the build artifacts to our repo as well, so the s2i/bin folder, dockerfile, default caddyfile, etc

Reply:
Hmmm, what's the chance that the existing repo will be maintained?

Reply:
I think the main risk would be if the repo was deleted entirely, as we can always lock our git ref to a specific revision to prevent accidentally taking breaking changes. Seems unlikely but who knows with the OCP 4 migration.